Austin_Marga_HW7_1.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>Be careful here mixing values and index. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_GreenhouseGases/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>